samples/azure-quantum/chemistry/QuantumPhaseEstimation/Molecule.ipynb

```diff
@@ -0,0 +1,497 @@
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "caffeine"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "102"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "caffeine.num_electrons"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2. Load Broombridge and simulate in Q#"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Preparing Q# environment...\n",
+      "."
+     ]
+    }
+   ],
+   "source": [
+    "import qsharp"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "data": {
+      "application/json": "{\"LastUpdated\":\"2021-05-07T14:16:55.767773-07:00\",\"IsCompleted\":false,\"Description\":\"Adding package microsoft.quantum.chemistry.jupyter\",\"Subtask\":null}",
+      "text/plain": [
+       "Adding package microsoft.quantum.chemistry.jupyter."
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from qdk.chemistry.broombridge import load_and_encode"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/json": "{\"LastUpdated\":\"2021-05-07T14:17:05.581833-07:00\",\"IsCompleted\":false,\"Description\":\"Reloading workspace\",\"Subtask\":null}",
+      "text/plain": [
+       "Reloading workspace."
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "qsharp.reload()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "encoded_data_caffeine = load_and_encode(\"../data/broombridge/caffeine.yaml\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3. Simulate energy of Caffeine molecule"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "from Microsoft.Quantum.Chemistry.QPE import GetEnergyQPE"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 3.75 ms, sys: 4.01 ms, total: 7.76 ms\n",
+      "Wall time: 875 ms\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "(-0.2699806186678728, 882.6628565016248)"
```

Comment:
In the demo I modified the code so that it returned only one number, the energy. This was consistent with other method, and didn't introduce extra numbers that are not referred to later